### PR TITLE
Fixed Built Doc Files from zipping.

### PR DESCRIPTION
**Pull request by alikasim72**
As the upload-artifact action already zips the file, the prior manual compression in the file is unnecessary and leads to "double compression" which is only uncompressed once during deployment stage. Hence, the lines of manual compression were omitted to fix this issue.

### CODE REVIEW

**Comment by mkundu1** [comment]
Uploading the doc artifact now takes [13 min](https://github.com/pyansys/pyfluent/actions/runs/3933547412/jobs/6727373010#step:15:20) instead of [35 sec](https://github.com/pyansys/pyfluent/actions/runs/3906690005/jobs/6675139815#step:16:20). Not zipping the doc artifact fixes the deployment, so we can zip it when we don't need to deploy.

**Comment by jorgepiloto** [comment]
Pinging here @MaxJPRey. I know PyAEDT run into a similar issue at some point but after checking their CI/CD pipelines it looks like they are no longer compressing the artifact.

I guess we could add an input parameter in pyansys actions to unzip the different artifacts.

**Comment by MaxJPRey** [comment]
> Pinging here @MaxJPRey. I know PyAEDT run into a similar issue at some point but after checking their CI/CD pipelines it looks like they are no longer compressing the artifact.
> 
> I guess we could add an input parameter in pyansys actions to unzip the different artifacts.

Agreed to expose the argument to unzip the content.

**Comment by jorgepiloto** [comment]
I just opened https://github.com/pyansys/actions/issues/143 to implement this.

**Comment by jorgepiloto** [comment]
Now that the decompress logic got added in the pyansys/actions, could you use the `decompress_artifact` input variable, @alikasim72?

**Comment by alikasim72** [comment]
Updated the code by adding back the compression code lines and setting the `decompress_artifact` to True.

**Review by mkundu1 — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -192,14 +192,15 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: HTML-Documentation-tag-${{ env.DOC_DEPLOYMENT_IMAGE_TAG }}
-          path: doc.zip
+          path: doc.zip
```
Comment (dnwillia-work) [inline .github/workflows/ci.yml:195]: maybe remove whitespace?

Comment (jorgepiloto) [inline .github/workflows/ci.yml:195]:
```suggestion
          path: doc.zip
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]

**Review by jorgepiloto — APPROVED** [review]

**Pull request merged**